js/interactive-guides/step-content.js

```diff
@@ -16,54 +16,108 @@ var stepContent = (function() {
 
   var setSteps = function(steps) {
     _steps = steps;
+    __createLinksToParentSteps();    
+  };
+
+  // Create link to parent step for each section so it can load the parent step when clicked on
+  var __createLinksToParentSteps = function() {
+    // Make links to parents
+    for(var i = 0; i < _steps.length; i++){
+      var step = _steps[i];
+      if(step.sections){
+        for(var j=0; j<step.sections.length; j++){
+          step.sections[j].parent = step;
+        }
+      }
+    }
   };
 
   var getCurrentStepName = function() {
     return currentStepName;
   };
 
-  // Hide the previous selected step content by looking for data-step attribute with the step name in it
+  // Hide the previous selected step content by looking for data-step attributes with the data-step not equal to the current step name
   var __hideContents = function() {
-    var stepToBeHidden = $("[data-step=" + currentStepName + "]");
-    stepToBeHidden.addClass("hidden");
+    var stepsToBeHidden = $("[data-step][data-step!=" + currentStepName + "]");
```

Comment:
just wondering do we need "[data-step]" if we already have "[data-step!=" + currentStepName + "]".

Reply:
Yes we do, if we just do data-step!=currentStepName it will literally get every tag on the page even if it doesn't have a data-step attribute. This is an AND clause

Reply:
Thanks for the explanation!